sql/sql_parse.cc

```diff
@@ -6206,31 +6206,84 @@ case SQLCOM_PREPARE:
                        first_level ? "MYSQL_AUDIT_QUERY_STATUS_END" :
                                      "MYSQL_AUDIT_QUERY_NESTED_STATUS_END");
 #endif /* !EMBEDDED_LIBRARY */
+#ifdef WITH_WSREP
+    int killed_errno;
```

Comment:
These variables are only set once. Please initialize them at the declaration and make them const.

Reply:
+1

Reply:
fixed